src/utils.js

```diff
@@ -87,7 +87,7 @@ ${
 }
 
 // Matches only the last occurrence of sourceMappingURL
-const innerRegex = /\s*[#@]\s*sourceMappingURL\s*=\s*([^\s'"]*)\s*/;
+const innerRegex = /\s*[#@]\s*sourceMappingURL\s*=\s*((?:[^\s'"\\]|\\[^n])*(?:\\n)?)\s*/;
```

Comment:
`^n`? Looks it is invalid

Reply:
The idea was to match everything except all whitespace, quotes and `\n`, but not a backslash on its own. It works like a negative lookahead but consuming characters. Another possible solution would be `.*?(?=[\s'"]|\\n)(?:\\n)?`, which actually looks a lot more readable (instead of `(?:[^\s'"\\]|\\[^n])*(?:\\n)?`)

And I've just noticed that with the current regex the match wouldn't stop when it encounters a whitespace or quote after a backslash, but it would have done before. I'm pushing a fix.

Also the test will fail, if it is in a multiline comment. Thanks for not letting this slip past!